src/test/java/com/google/sps/MatchServletTest.java

```diff
@@ -150,8 +153,8 @@ private String doGetHelper(HttpServletRequest request, HttpServletResponse respo
 
     //count number of opening brackets in result string, which is equal to number of matches
```

Comment:
nit: This comment looks out of date, now that you use the array length (instead of counting the braces yourself).